projects/tesseract-ocr/build.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -eu
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+cd $SRC
+tar zxf leptonica-1.77.0.tar.gz
+cd leptonica-1.77.0
+./configure
+make -j$(nproc)
+make install
+ldconfig
+
+cd $SRC/tesseract
+./autogen.sh
+CXXFLAGS="$CXXFLAGS -D_GLIBCXX_DEBUG" ./configure --disable-graphics
+make -j$(nproc)
+
+cd $SRC/tesseract-ocr-fuzzers
+
+mv $SRC/tessdata $OUT
+export TESSDATA_PREFIX=`realpath $OUT/tessdata`
+
+$CXX $CXXFLAGS \
+    -DTESSDATA_PREFIX="\"$TESSDATA_PREFIX\"" \
+    -I $SRC/tesseract/src/api \
+    -I $SRC/tesseract/src/ccstruct \
+    -I $SRC/tesseract/src/ccmain \
+    -I $SRC/tesseract/src/ccutil \
+     $SRC/tesseract-ocr-fuzzers/fuzzer-api.cpp -o $OUT/fuzzer-api \
+     $SRC/tesseract/src/api/.libs/libtesseract.a \
+     /usr/local/lib/liblept.a \
+     /usr/lib/x86_64-linux-gnu/libtiff.a \
+     /usr/lib/x86_64-linux-gnu/libpng.a \
+     /usr/lib/x86_64-linux-gnu/libjpeg.a \
+     /usr/lib/x86_64-linux-gnu/libjbig.a \
+     /usr/lib/x86_64-linux-gnu/liblzma.a \
+     /usr/lib/x86_64-linux-gnu/libz.a \
```

Comment:
Hm that's weird, doesn't `-lz` work? We have bunch of other projects depending on this and it works just fine: https://github.com/google/oss-fuzz/search?l=Shell&q=-lz&type=

Reply:
Should that enforce static linking for those libraries?

Reply:
Fixed.